src/stylesheets/elements/_tags.scss

```diff
@@ -1,11 +1,10 @@
 .usa-tag {
-  @include u-bg('base-dark');
   @include u-font('sans', '2xs');
-  @include u-margin-right(0.5);
-  @include u-padding-x(1);
-  @include u-padding-y(1px);
   @include u-radius('sm');
```

Comment:
Should this be `border-radius: units(2px);` or the mixin?

Reply:
Is there a border radius function a la `border-radius: radius(sm);`?

Reply:
no, but there should be. made an issue here: https://github.com/uswds/uswds/issues/2723

Reply:
Added the function d0a8c88, seems to work.